cli/src/merge_tools/mod.rs

```diff
@@ -97,8 +103,17 @@ pub enum ConflictResolveError {
          see the exact invocation)."
     )]
     EmptyOrUnchanged,
-    #[error("Backend error")]
+    #[error(transparent)]
     Backend(#[from] jj_lib::backend::BackendError),
+    #[error(transparent)]
+    Io(#[from] std::io::Error),
+}
+
+#[derive(Debug, Error)]
+#[error("Stopped due to error after resolving {resolved_count} conflicts")]
+pub struct MergeToolPartialResolutionError {
+    pub source: Box<ConflictResolveError>,
```

Comment:
nit: it's probably better to extract error of "conflict resolution"-type (i.e. split `ExternalToolError` and `BuiltinToolError`), but that will require large refactoring.

(no action required)